### PR TITLE
[IA-2920] [IA-2914] [IA-2917] PPW fixes for Galaxy and runtime environment variables

### PR DESCRIPTION
**Pull request by rtitle**
JIRAs:
https://broadworkbench.atlassian.net/browse/IA-2914
https://broadworkbench.atlassian.net/browse/IA-2920
https://broadworkbench.atlassian.net/browse/IA-2917

This PR fixes Galaxy creation under PPW, and correctly sets environment variables for all runtime types under PPW.

This PR does not depend on the PPW feature branch; it can merge to dev anytime.

Tested manually using a fiab.

<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>

### CODE REVIEW

File: src/pages/workspaces/workspace/WorkspaceContainer.js [hunk 1]
```diff
@@ -320,7 +320,7 @@ export const wrapWorkspace = ({ breadcrumbs, activeTab, title, topBarContent, sh
       }, [
         workspace && h(WrappedComponent, {
           ref: child,
-          workspace, refreshWorkspace, refreshRuntimes, runtimes, persistentDisks, galaxyDataDisks,
+          workspace, refreshWorkspace, refreshRuntimes, refreshApps, runtimes, persistentDisks, galaxyDataDisks,
```
Comment (rtitle) [inline src/pages/workspaces/workspace/WorkspaceContainer.js:323]: Unrelated to PPW, but I think this change fixes https://broadworkbench.atlassian.net/browse/IA-2917. Previously `refreshApps` was not being passed to the `Notebooks` child component, causing the `Cannot read property 'apply' of undefined` error when the function was explicitly called after Galaxy creation.

File: src/components/GalaxyModal.js [hunk 2]
```diff
@@ -40,9 +40,10 @@ const titleId = 'galaxy-modal-title'
 
 export const GalaxyModalBase = Utils.withDisplayName('GalaxyModal')(
   ({
-    onDismiss, onSuccess, apps, galaxyDataDisks, workspace, workspace: { workspace: { namespace, bucketName, name: workspaceName } },
+    onDismiss, onSuccess, apps, galaxyDataDisks, workspace, workspace: { workspace: { namespace, bucketName, name: workspaceName, googleProject } },
     isAnalysisMode = false
   }) => {
+
```
Comment (kyuksel) [inline src/components/GalaxyModal.js]: This causes a linting error:

```
src/components/GalaxyModal.js
  Line 45:9:  Block must not be padded by blank lines  padded-blocks
```

One easy way to fix all linting errors is to run

```
yarn run eslint src --fix
```

on the command line from the root of the `terra-ui` repo.

File: src/pages/workspaces/workspace/WorkspaceContainer.js (hunk 1 above)
Comment (kyuksel) [reply src/pages/workspaces/workspace/WorkspaceContainer.js:323]: Thanks for catching this!

**Review by kyuksel — APPROVED** [review]
LGTM with a linter error notice inline  👍🏼 
Way to do a [hat-trick](https://en.wikipedia.org/wiki/Hat-trick) with a +13-6 PR  🚀

**Comment by rtitle** [comment]
I tested backwards compatibility against `dev` too, looks good. 👍

**Review by petesantos — APPROVED** [review]

**Pull request merged**